cassandra/src/main/scala/org/apache/zeppelin/cassandra/DisplaySystem.scala

```diff
@@ -507,11 +558,15 @@ object MetaDataConverter {
         .map(c => new ColumnDetails(c.columnMeta.getName, StaticColumn, c.columnMeta.getType))
     }
 
-    protected def convertClusteringColumns(columns: List[ColumnMetaWrapper], orders: List[DriverClusteringOrder]): List[ColumnDetails] = {
+    protected def convertClusteringColumns(
+      columns: List[ColumnMetaWrapper],
+      orders: List[DriverClusteringOrder]
+    ): List[ColumnDetails] = {
```

Comment:
What I mean is "columns" should located in upper line when the length of that case is less than 100. If it is greater than 100, "columns" and "orders" are in the same line. vertical align among parameters is not recommended.